services/QuillLMS/app/services/classlink_integration/auth_credential_saver.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module ClasslinkIntegration
+  class AuthCredentialSaver < ApplicationService
+    JWT_PASSWORD = nil
+    JWT_SHOULD_VALIDATE = false
+    attr_reader :code
+
+    def initialize(code:)
+      @code = code
+    end
+
+    def run
+      fetch_credentials
+        .then { |cs| AuthCredentialSaver.format_credentials(cs) }
```

Comment:
`cs` and `fcs` feel a little cryptic.  Maybe use a longer temp variable `creds` and `formatted creds` for readability.